src/modules/package.rs

```diff
@@ -68,6 +68,23 @@ fn get_node_package_version(context: &Context, config: &PackageConfig) -> Option
     Some(formatted_version)
 }
 
+fn get_jsr_package_version(context: &Context, config: &PackageConfig) -> Option<String> {
+    let json_content = context
+        .read_file_from_pwd("jsr.json")
+        .and_then(|contents| json::from_str(&contents).unwrap_or_default())
+        .or_else(|| json::from_str(&context.read_file_from_pwd("deno.json")?).unwrap_or_default())
+        .or_else(|| {
+            jsonc_parser::parse_to_serde_value(
+                &context.read_file_from_pwd("deno.jsonc")?,
+                &Default::default(),
+            )
+            .unwrap_or_default()
+        })?;
```

Comment:
This should probably cover `jsr.jsonc`. I would also prefer if this code use`.find()` with an iterator over a list of the filenames.